sass/routes/_settings.scss

```diff
@@ -356,13 +356,18 @@
                     .theme-elements__body {
                         @include border-radius(0 0 3px 3px);
                         @include legacy-pie-clearfix;
-                        @include alpha-property(background-color, $white, .05);
-                        display: none;
+                        @include alpha-property(background-color, $white, 0.05);
+
                         margin: 0 20px 0 0;
-                        padding: 24px 0 0 24px;
+                        padding: 0 0 0 24px;
+                        transition: all 0.5s ease-in-out;
```

Comment:
Could you shorten this slightly to 0.4s? That's the default duration used by jquery's `slideToggle`

Reply:
Done :)